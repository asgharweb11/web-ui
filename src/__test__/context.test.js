import React from "react"
import renderer from "react-test-renderer"
//--------- Component
import Context from "../context"


test('Component : Constract/footer.js', () => {
    const container = renderer.create(<Context />)

    let tree = container.toJSON();
    expect(tree).toMatchSnapshot()
    console.log(Context)

})
