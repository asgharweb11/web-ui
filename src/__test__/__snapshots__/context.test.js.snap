// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component : Constract/footer.js 1`] = `null`;
